### DISCUSSION TITLE
Unable to download HA-floorplan from HACS

### DISCUSSION THREAD

**Discussion by WuodDala** [ExperienceLovelace/ha-floorplan#327] [Q&A] [answered]
I have been having an issue downloading ha-floorplan.
I got the error below from the log. I am not that technical so, maybe there is something i am missing

![image](https://user-images.githubusercontent.com/126223556/221654511-03e6ba71-1f23-420a-a042-d10307f45ccf.png)
 This is the only item from HACS i have been unable to download. i have tried multiple other items and they download just fine.

HA Version
![image](https://user-images.githubusercontent.com/126223556/221654867-b83a9c86-b25f-4789-9dce-2f8d36aa16aa.png)

HACS version
![image](https://user-images.githubusercontent.com/126223556/221654952-06bcf785-ae30-4e82-a334-5fc961ded7c0.png)

ha-floorplan folder is never created
![image](https://user-images.githubusercontent.com/126223556/221655115-67cc5bc9-0580-4b13-8dda-aaf88e8b3124.png)

**Comment by OzGav** [comment, marked as answer]
You can try and put this in your browser window and see what happens:
https://raw.githubusercontent.com/ExperienceLovelace/ha-floorplan/1.0.35/dist/floorplan.js

You should see a bunch of text which is the javascript file. However at the end of the day if HACS can't pull it down there isn't much we can do. You can just use the manual install method and check back occasionally to see if there are any updates. A new version is being worked on at the moment.

**Reply by WuodDala** [reply]
New update fixed it!
